_layouts/default.html

```diff
@@ -10,12 +11,14 @@
     
     <main class="container">
       {{ content }}
-      <img src="/assets/media/HAAM_Blue.svg" class="center" width="60px">
+    <!-- code below includes how to add hover text.-->
+    <!--img src="/assets/media/HAAM_Blue.svg" class="center" width="60px" title="The HAAM logo was designed by Irene Rainer."-->
+    <img src="/assets/media/HAAM_Blue.svg" class="center" width="60px">
+    <h1 style="font-size: 0.4rem; color: #000000"; class="center">The HAAM logo was designed by Irene Rainer.</h1>
```

Comment:
```suggestion
    <h1 style="font-size: 0.4rem"; class="center">The HAAM logo was designed by Irene Rainer.</h1>
```
I think the colour does not need to be set here. 
a) I think the default should already be black and
b) If we change the defaults through CSS theming later on, it will be easier to keep everything consistent if we don't hard-code the colour.